fuzzing/contracts/contract.go

```diff
@@ -66,3 +115,24 @@ func (c *Contract) CompiledContract() *types.CompiledContract {
 func (c *Contract) Compilation() *types.Compilation {
 	return c.compilation
 }
+
+// PropertyTestMethods returns the methods that are property tests.
+func (c *Contract) PropertyTestMethods() []abi.Method {
+	return c.propertyTestMethods
+}
+
+// OptimizationTestMethods returns the methods that are optimization tests.
+func (c *Contract) OptimizationTestMethods() []abi.Method {
+	return c.optimizationTestMethods
+}
+
+// AssertionTestMethods returns the methods that are assertion tests.
+func (c *Contract) AssertionTestMethods() []abi.Method {
+	return c.assertionTestMethods
+}
+
+func (c *Contract) AddTestMethods(assertion, property, optimization []abi.Method) {
+	c.assertionTestMethods = assertion
+	c.propertyTestMethods = property
+	c.optimizationTestMethods = optimization
+}
```

Comment:
I am going to expose `AssertionTestMethods` and so on because if you are providing a way to read and write to it, you might as well directly expose it. Additionally, you can then individually set each one instead of being forced to do it via `AddTestMethods.`